api/swagger.yml

```diff
@@ -3156,6 +3156,11 @@ paths:
           schema:
             type: string
             format: date-time
+        - in: query
+          name: stop_at
+          description: a reference to stop at
+          schema:
+            type: string
```

Comment:
Very late to the party :)
1. How does this new parameter interact with `since`? What happens if they both are given? worth documenting this behavior.
2. Given `since`, what capabilities does `stop_at` grant you on top of that? I guess that if you have the commit at hand you can use `since` with its creation date, right?

Reply:
<img width="1303" alt="Screenshot 2023-12-31 at 12 05 47" src="https://github.com/treeverse/lakeFS/assets/43949240/6c56b636-70c9-4373-afd2-23f2e48722b7">

@itaiad200 Theoretically you're right, but the commit log may act funny if we don't run lakeFS server from machines with the same timezone.

Reply:
Wait, what?!  This is a timezone bug.  I hope it's just in the GUI.  Please open an issue.